aiida/manage/external/postgres.py

```diff
@@ -117,22 +116,45 @@ def drop_dbuser(self, dbuser):
         """
         self.execute(_DROP_USER_COMMAND.format(dbuser))
 
-    def check_dbuser(self, dbuser):
-        """Looks up if a given user already exists, prompts for using or creating a differently named one.
+    def find_new_dbuser(self, start_from='aiida'):
+        """Find database user that does not yet exist.
 
-        :param str dbuser: Name of the user to be created or reused.
-        :returns: tuple (dbuser, created)
+        Generates names of the form "aiida_1", "aiida_2", etc. until it finds a name that does not yet exist.
+
+        :param str start_from: start from this name
+        :returns: dbuser
+        """
+        dbuser = start_from
+        i = 0
+        while self.dbuser_exists(dbuser):
+            echo.echo_warning(f'Database user "{dbuser}" already exists.')
+            i = i + 1
+            dbuser = f'{start_from}_{i}'
```

Comment:
If people run `verdi quicksetup` a lot, this risks printing a lot of warnings. Should we maybe just use a uuid as suffix?

Reply:
This warning will only be printed, when (a) the database user already exists and (b) the password for this user in the aiida config is incorrect.

However, I agree that there is no need to have the warning inside the loop - it's good enough to know that the dbuser the user asked for was taken. I will fix that.